content/en/docs/ops/integrations/prometheus/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Prometheus
+description: Information on how to integrate with Prometheus.
+weight: 20
+keywords: [integration,prometheus]
+---
+
+[Prometheus](https://prometheus.io/) is an open source monitoring system and time series database. You can use Prometheus with Istio to record metrics that track the health of Istio and of applications within the service mesh. These metrics can be visualized using tools like [Grafana](/docs/ops/intgrations/grafana/) and [Kiali](/docs/tasks/observability/kiali/).
+
+## Configuration
+
+In an Istio mesh, each component will expose an endpoint that emits metrics. Prometheus works by scraping these endpoints and collecting the results. This is is configured through the [Prometheus configuration file](https://prometheus.io/docs/prometheus/latest/configuration/configuration/) that controls which endpoints to query, the port and path to query, TLS settings, and more.
```

Comment:
This is is -> This is

Reply:
This is configured via the [Prometheus configuration file](https://prometheus.io/docs/prometheus/latest/configuration/configuration/) which controls settings for endpoints to query, ports/path to use, etc.